hbase_sql/src/main/java/com/splicemachine/ddl/ZooKeeperDDLWatchChecker.java

```diff
@@ -159,4 +189,30 @@ public void killDDLTransaction(String key){
        zkClient.deleteChangeNode(key);
     }
 
+
+    @Override
+    public void assignDDLDemarcationPoint(long txnId) throws Exception {
+        String rootPath = SIDriver.driver().getConfiguration().getSpliceRootPath();
+        String demarcationPath = rootPath+HConfiguration.DDL_PATH+"/"+id;
+        byte[] data = ZkUtils.getData(demarcationPath);
+        long demarcationPoint = data != null ? Bytes.toLong(data) : 0;
+        if (txnId > demarcationPoint) {
```

Comment:
I don't think we need this check, we already made sure it's the most recent one in `DDLWatchRefresher.assignDDLDemarcationPoint` We can write blindly into ZK I'd say

Reply:
Right, this check is duplicate. Removed.